src/commands/activity/activity.ts

```diff
@@ -0,0 +1,77 @@
+import { discordClient, robloxClient, robloxGroup } from '../../main';
+import { CommandContext } from '../../structures/addons/CommandAddons';
+import { Command } from '../../structures/Command';
+import { PartialUser, User, GroupMember } from 'bloxy/dist/structures';
+import { getLinkedRobloxUser } from '../../handlers/accountLinks';
+import { config } from '../../config';
+import {
+    getActivityEmbed,
+    getInvalidRobloxUserEmbed,
+    getRobloxUserIsNotMemberEmbed,
+    getUnexpectedErrorEmbed,
+    getUserInfoEmbed,
+} from '../../handlers/locale';
+import { provider } from '../../database/router';
+
+class ActivityCommand extends Command {
```

Comment:
Why not just combine this with the info command? Having another one seems unnecessary.

Reply:
okay